recipes/android-ndk/all/conanfile.py

```diff
@@ -93,9 +94,18 @@ def _clang_triplet(self):
         return f"{arch}-linux-{abi}"
 
     @property
-    def _ndk_version(self):
-        assert self.version[0] == "r"
-        return self.version[1:]
+    def _ndk_version_major(self):
+        match = re.search(r"r(\d+)\w?", self.version)
```

Comment:
You can combine these regexes in one:
```python
@property
def _ndk_major_minor(self):
    match = re.search(r"r(\d+)(\w?)", self.version)
    assert match
    major, minor = match.groups()
    assert major
    return int(major), minor if minor else "a"
@property
def _ndk_version_major(self):
    return self._ndk_major_minor[0]
@property
def _ndk_version_minor(self):
    return self._ndk_major_minor[1]
```
Cleaner code, imho.

Reply:
Good recommendation. Am going to implement that. Thank you!

Reply:
Implemented, and tested on Windows and Linux just for good measure.